keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -881,6 +887,73 @@ Other components:
 * check how the change of meaning of resource requests influence other
   Kubernetes components.
 
+### Instrumentation
+
+The kubelet will record the following metrics:
+
+#### `kubelet_container_resize_requests_total` 
+
+This metric tracks the total number of resize requests observed by the Kubelet, counted at the container level.
```

Comment:
How would this be implemented? Would we count it at update time? Unless it's recorded by the apiserver, I think it's possible that multiple resizes in rapid succession could be counted together, but I'm not sure we care. See https://github.com/kubernetes/kubernetes/blob/92de70895830ea1a9c2c6554bdab4cbee7ce867d/pkg/kubelet/pod/pod_manager.go#L158 for how this is done for ephemeral containers.